_articles/fixing-drives.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: article
+title: Mount And Repair File Systems
+description: >
+    How to mount and repair common file system issues.
+keywords:
+  - Repair
+  - File System
+  - Ubuntu
+  - Pop
+hidden: false
+section: software-applications
+
+---
+
+## Problem
+
+File Systems can be damaged from events like unexpected power loss or unsafe removal of drive(s). This is where you would run a File System repair tool on your system. There are a few ways to do this and we'll list them below:
+
+## Solution
+
+If you are not able to boot into the installed OS due to this issue we can boot into a [live disk](http://support.system76.com/articles/live-disk/), [mount then chroot](https://support.system76.com/articles/bootloader/) into the installed OS to go though these steps. 
```

Comment:
Grammar:

"mount the drive, then chroot into the installed OS. Once chrooted, follow these steps:"

Reply:
Are you sure?

Reply:
Yes. Mount [link] and chroot [link] should each have hyperlinks to those specific parts of the article. And the sentence is clearer in two parts.

Previously it was a run-on sentence.